components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -1012,11 +1012,13 @@
 <micro_nicons         microphys="mg2">  0.1D6       </micro_nicons>
 
 <!-- P3 specific namelist variables -->
-<micro_p3_lookup_dir> atm/cam/physprops </micro_p3_lookup_dir>
-<micro_p3_tableversion> 4 </micro_p3_tableversion>
-<micro_aerosolactivation > .true.  </micro_aerosolactivation>
-<micro_subgrid_cloud     > .true.  </micro_subgrid_cloud>
-<micro_tend_output       > .true.  </micro_tend_output>
+<micro_p3_lookup_dir	    > atm/cam/physprops </micro_p3_lookup_dir>
```

Comment:
Is there a reason for the spaces inside these tags? Does this ever cause trouble with XML parsers?